Detectors/CTP/reconstruction/src/RawDataDecoder.cxx

```diff
@@ -266,8 +266,7 @@ int RawDataDecoder::decodeRaw(o2::framework::InputRecord& inputs, std::vector<o2
         ret = addCTPDigit(linkCRU, rdhOrbit, diglet, pldmask, digitsMap);
       }
     }
-    // if ((remnant.count() > 0) && stopBit) {
-    if (remnant.count() > 0) {
+    if ((remnant.count() > 0) && stopBit) {
```

Comment:
Hi @lietava did you also test this with healthy raw data? Likely I am wrong, but just by looking at the code it seems that in case the `stopBit == true` then you don't have any payload in which case also `remnant` will stay at `0` no? In that case no digits would be written out. I did not run it myself and as said just looked at the code, but I wanted to make sure